examples/benchmark/benchmark.py

```diff
@@ -1,52 +1,47 @@
 import os
 import time
 import siliconcompiler
-from siliconcompiler.flows import asicflow
 
 # Setting up the experiment
-
-
 def main():
     rootdir = os.path.dirname(__file__)
 
     examples = [{'heartbeat': 1000},
-                {'picorv32': 1000},
```

Comment:
The `picorv32.v` is not delivered with siliconcompiler, maybe it's better to leave it out so that the example runs straight after cloning.

Reply:
I'll take a look at this, there is a method to get the picorv32 source files via the package source handling we introduced a few months ago.

Reply:
I added the package mechanism as it used in `examples/picorv32`